.github/workflows/cd-dashboard.yml

```diff
@@ -45,6 +47,25 @@ jobs:
       - name: Build and Push Container
         working-directory: ./packages/dashboard
         run: |
+          cat << EOF > .env
```

Comment:
you create this file here... but where do you use it?

Reply:
It's used for gcloud build submit, it references .env that is in gcp cloud bucket cloud build for nextjs client